static/app/views/issueDetails/streamline/groupActivityIcons.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  IconAdd,
+  IconCheckmark,
+  IconClose,
+  IconDelete,
+  IconEdit,
+  IconFile,
+  IconFire,
+  IconFlag,
+  IconGraph,
+  IconLock,
+  IconMute,
+  IconNext,
+  IconPlay,
+  IconPrevious,
+  IconRefresh,
+  IconUnsubscribed,
+  IconUser,
+} from 'sentry/icons';
+import {GroupActivityType} from 'sentry/types/group';
+
+export const GroupActivityTypeIconMapping: Record<GroupActivityType, React.ReactNode> = {
```

Comment:
This is my attempt at getting somewhat unique icons for each different activity type that somewhat matches the activity they are describing. will happily take suggestions for improvements here.